extensions/twenty/src/create-people.tsx

```diff
@@ -0,0 +1,169 @@
+import { Action, ActionPanel, Form, showToast, Toast, useNavigation } from "@raycast/api";
+import { FormValidation, useForm } from "@raycast/utils";
+import axios, { isAxiosError } from "axios";
+import { useState } from "react";
+import { useAuthHeaders } from "./hooks/use-auth-headers";
+import { useGetCompanies } from "./hooks/use-company";
+import ListPeople from "./list-people";
+import { Person } from "./types";
+import ErrorView from "./error-view";
+
+interface CreatePersonFormProps {
+  firstName: string;
+  lastName: string;
+  primaryEmail: string;
+  primaryPhoneNumber?: string;
+  linkedinLink?: string;
+  jobTitle?: string;
+  city?: string;
+  companyId?: string;
+}
+
+export default function CreatePersonForm() {
+  const { push } = useNavigation();
+  const { companies, isLoading, error, hasMore, loadMore } = useGetCompanies();
+  const [creationIsLoading, setCreationIsLoading] = useState(false);
+
+  if (error) {
+    return <ErrorView error={error} />;
+  }
```

Comment:
When the API Key is invalid, this return statement causes the extension to crash since the `useForm` hook is not rendered. Move this to AFTER the `useForm` to fix 👀.